src/site/content/en/blog/test-post/index.md

```diff
@@ -513,3 +601,74 @@ porta dolor erat, vel molestie dolor posuere in. Nam vel elementum augue. Nam
 quis enim blandit, posuere justo dignissim, scelerisque diam. Fusce aliquet urna
 ac blandit ullamcorper. Proin et semper nibh, sit amet imperdiet velit. Morbi at
 quam sem.
+
+{% Details %}
+
+{% DetailsSummary %}
+Details _summary_
+{% endDetailsSummary %}
+
+This is an optional preview. Make your preview text match the first paragraph
+of your panel text.
+
+Lorem ipsum [dolor sit amet](#), consectetur adipiscing elit. Proin dictum a massa
+sit amet ullamcorper. `Suspendisse` auctor ultrices ante, nec tempus nibh varius
+at.
+
+{% endDetails %}
+
+{% Details %}
+
+{% DetailsSummary %}
+Details _summary_
+{% endDetailsSummary %}
+
+Lorem ipsum [dolor sit amet](#), consectetur adipiscing elit. Proin dictum a massa
+sit amet ullamcorper. `Suspendisse` auctor ultrices ante, nec tempus nibh varius
+at.
+
+{% endDetails %}
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dictum a massa
+sit amet ullamcorper. Suspendisse auctor ultrices ante, nec tempus nibh varius
+at. Cras ligula lacus, porta vitae maximus a, ultrices a mauris. Vestibulum
+porta dolor erat, vel molestie dolor posuere in. Nam vel elementum augue. Nam
+quis enim blandit, posuere justo dignissim, scelerisque diam. Fusce aliquet urna
+ac blandit ullamcorper. Proin et semper nibh, sit amet imperdiet velit. Morbi at
+quam sem.
+
+{% Instruction 'remix', 'ol' %}
+{% Instruction 'preview', 'ol' %}
+
+To explain how to open the Glitch console, use the `console` argument:
+
+{% Instruction 'console', 'ol' %}
+
+To explain how to create a new file in a Glitch, use the `create` argument:
+
+{% Instruction 'create', 'ol' %}
+
+To explain how to view a Glitch's source code, use the `source` argument:
+
+{% Instruction 'source' %}
+
+If users are reloading an app, use the `reload-app` argument:
+
+{% Instruction 'reload-app' %}
+
+If users are reloading a traditional web page, use the `reload-page` argument:
+
+{% Instruction 'reload-page' %}
+
+If users are reloading a page for the purpose of profiling,
+use the `start-profiling` argument:
+
+{% Instruction 'start-profiling' %}
```

Comment:
One of the `li` is off. Probably should set `li` to use `display: inline;`

![image](https://user-images.githubusercontent.com/11811422/151467372-56c14af6-581b-47ee-8dcf-cfc469fdecfc.png)

Reply:
Yes, this is why these elements were added to test post, to catch all regressions we have elswhere in the code.